chart/Chart.yaml

```diff
@@ -11,6 +11,7 @@ description: |
 name: rancher-externalip-webhook
 version: %VERSION%
 appVersion: %APP_VERSION%
+kubeVersion: < 1.22.0
```

Comment:
still need 
```suggestion
kubeVersion: < 1.22.0-0
```

Reply:
Actually this doesn't matter, we don't do anything with kube alphas typically. I was thinking rancher version.